### DISCUSSION TITLE
news/20230720/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#221] [Website Discussions]
# news/20230720/

I've been cutting all day with Final Cut Pro 10.6.7.

https://fcp.cafe/news/20230720/

**Comment by dmetz01** [comment]
Interestingly (or not), I was able to fix the Placeholder generator by opening a copy in Motion, and changing focus of the rig's opacity toggle from the text item to its group container.  The new copy functions properly in FCP.  This doesn't fix the original, but may be helpful in diagnosing the problem.

I built another generator some years ago that was also broken with this update - in that case the Colorize filter was applied to the source image for a particle emitter.  When 'colorize' was selected, all particles disappeared instead of taking on the new colors.  Moving the filter from the source image to the emitter itself has fixed that one.

**Comment by dmetz01** [comment]
One other note - I have other generators that utilize the same opacity toggle, and they all work perfectly.  So it isn't specific to opacity - I think this has more to do with the parameter's target, and its hierarchy within the Motion layers/stack/whatever.
